routes/trigger/trigger_api.go

```diff
@@ -43,6 +45,22 @@ func New(controller decomposer_controller.IController, database database.IContro
 	return &instance
 }
 
+func MergeVariablesInPlaybook(playbook *cacao.Playbook, body []byte) {
```

Comment:
We should add a function that checks that all injected variables from the trigger API payload actually exist in the playbook variables, otherwise send back a bad request and not execute further